### PR TITLE
Update tests for ewatercycle 2.3.1

### LINKED ISSUE #8: Update tests for ewatercycle v2.1.0
With the changes in ewatercycle v2.1.0, the tests fail as they compare the entire output of the forcing class object.
I.e. `unexpected filenames={} in forcing object`.

The tests should be updated to reflect the new behavior.

### PR DESCRIPTION
**Pull request by BSchilperoort**
Closes #8

I re-rendered the forcing notebook.

The model.ipynb notebook ran without issues.

### CODE REVIEW

File: tests/test_forcing.py [hunk 1]
```diff
@@ -169,8 +171,7 @@ def test_saved_yaml_content(self, forcing, tmp_path):
 
     def test_saved_yaml(self, forcing, tmp_path):
         saved_forcing = WflowForcing.load(tmp_path)
-        # shape should is not included in the yaml file
-        forcing.shape = None
+        forcing.shape = forcing.directory / "Rhine.shp"
```
Comment (BSchilperoort) [inline tests/test_forcing.py:174]: shape is now copied upon saving, so this behavior changed.

File: src/ewatercycle_wflow/forcing.py [hunk 2]
```diff
@@ -28,6 +28,7 @@ class WflowForcing(DefaultForcing):
         Inflow (str) = None: Variable name of inflow data in input file.
     """
 
+    filenames: dict[str, str] = {}
```
Comment (BSchilperoort) [inline src/ewatercycle_wflow/forcing.py:31]: I chose to leave this empty as wflow (old python version) requires a single netcdf for multiple variables. It had to be included though to follow the parent class.

Comment (sverhoeven) [reply src/ewatercycle_wflow/forcing.py:31]: weird that extending needs you to repeat the properties, oh well.

**Review by sverhoeven — APPROVED** [review]
Looks good to me

File: src/ewatercycle_wflow/forcing.py (hunk 2 above)
Comment (BSchilperoort) [reply src/ewatercycle_wflow/forcing.py:31]: Yeah not sure, didn't have to do this with Hype. I'll double check...

Comment (BSchilperoort) [reply src/ewatercycle_wflow/forcing.py:31]: Oh it runs fine without that line now :man_facepalming:

**Comment by BSchilperoort** [comment]
Didn't need to bump the version: I only modified the tests.

**Pull request merged**